### ISSUE TITLE
Start command should handle OpenShift versions with and without 'v' prefix

### ISSUE THREAD

**Issue by hferentschik** [minishift/minishift#410]
_minishift start --openshift-version \<version\>_ should work with or without 'v' prefix, eg v1.3.1 or 1.3.1. Or at least it should give a clear error message that the prefix is needed.

**Comment by gbraad** [comment]
at the moment I have amazed me at the same for the origin releases, as the containers are tagged with a prefixed `v` for the tag. At the moment the pulling from Docker hub would result in a Metadata unknown, and this is likely also shown in minishift.

However, should we else allow a subset of supported openshift versions (as in curated, and for error purposes), or just check for the prefix? WDYT? I prefer not to have a curated list, but I could understand that this might be helpful for CDK.

**Comment by gbraad** [comment]
This could be a trivial fix... and especially with the version check the behaviour is misleading:

```
C:\Users\gbraad\Downloads>minishift.exe start --openshift-version=1.4.0
Minishift does not support Openshift version 1.4.0 .You need to use a version >=v1.4.1

C:\Users\gbraad\Downloads>minishift.exe start --openshift-version=1.4.1
Starting local OpenShift cluster using 'hyperv' hypervisor...
-- Checking OpenShift client ... OK
-- Checking Docker client ... OK
-- Checking Docker version ... OK
-- Checking for existing OpenShift container ... OK
-- Checking for openshift/origin:1.4.1 image ...
   Pulling image openshift/origin:1.4.1
FAIL
   Error: error pulling Docker image openshift/origin:1.4.1
   Caused By:
     Error: Tag 1.4.1 not found in repository docker.io/openshift/origin
Error starting the cluster.
```

**Comment by hferentschik** [comment]
What I had in mind is definitely a trivial fix. For now I was thinking of transparently adding the 'v' if it is not specified.

**Comment by hferentschik** [comment]
Merged via pull request #972
